docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+
+### Emulate a Device
+
+Specify the target device with the `device` parameter.
+
+```sh
+testcafe "chrome:emulation:device=iphone X" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:emulation:device=iphone X')
+    .run();
+```
+
+### Emulate a Screen Size
+
+You can specify parameters like `width`, `height`, or `orientation` to configure the device emulator.
+
+```sh
+testcafe "chrome:emulation:width=100;height=200;mobile=true;orientation=vertical;touch=true" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:emulation:width=100;height=200;mobile=true;orientation=vertical;touch=true')
+    .run();
+```
+
+### Use Emulation in Headless Mode
+
+You can combine both device emulation and headless mode.
+
+```sh
+testcafe "chrome:headless:emulation:device=iphone X;cdpPort=9223" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:headless:emulation:device=iphone X;cdpPort=9223')
+    .run();
+```
+
+### Use Emulation in Portable Chrome
+
+To enable device emulation in a portable Chrome, use the [browser alias](#locally-installed-browsers). The `path:` prefix does not work in this case.
+
+```sh
+testcafe "chrome:d:\chrome_portable\chrome.exe:emulation:device=iphone X" tests/test.js
+```
+
+### Substitute a User Agent
+
+Use the `userAgent` parameter to substitute a user agent string.
+
+> Important! TestCafe relies on the user agent string to emulate the browser behavior. Tests are not guaranteed to run correctly if you provide a user agent that is invalid or [not supported by TestCafe](#browser-support).
+
+You need to escape special characters in the user agent string when you specify it in the command line. These characters include:
+
+* `\` (backslash)
+* `'` (single quote)
+* `"` (double quote)
+* `,` (comma)
+* `;` (semicolon)
+* `:` (colon)
+
+The way to escape special characters depends on the shell you use. You also need to escape semicolons from the TestCafe argument parser with an additional backslash.
+
+The following examples show user agent strings escaped for `bash` and `PowerShell`.
+
+**bash**
+
+In `bash`, put a dollar sign before the argument to allow single quotes. Escape special characters with a backslash and use a double backslash for semicolons to escape them from the TestCafe argument parser:
+
+```sh
+testcafe $'chrome:emulation:userAgent=\'Mozilla/5.0 (Windows NT 10.0\\; Win64\\; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36\'' test.js
+```
+
+**PowerShell**
+
+In `PowerShell`, escape special characters with a single quote and use a backslash for semicolons to escape them from the TestCafe argument parser:
```

Comment:
Can we say:
In `PowerShell`, escape special characters with a single quote and use a backslash for semicolons - to escape them from the TestCafe argument parser: